src/utilities/lifecycle_service_client.cpp

```diff
@@ -86,7 +86,7 @@ bool LifecycleServiceClient::configure()
 bool LifecycleServiceClient::shutdown()
 {
   return change_state(rclcpp_lifecycle::Transition(lifecycle_msgs::msg::Transition::
-           TRANSITION_DEACTIVATE));
+           TRANSITION_ACTIVE_SHUTDOWN));
```

Comment:
This seems unrelated? Could this be a different PR?

Reply:
Yep, just noticed this when writing the tests. Separate PR is here: https://github.com/aws-robotics/ROS2-SecTest/pull/15

Reply:
#15 is merged